g/gnome-system-monitor/stone.yaml

```diff
@@ -18,6 +18,7 @@ builddeps   :
     - binary(lld)
     - binary(msgfmt)
     - binary(python3)
+    - pkgconfig(catch2-with-main)
```

Comment:
This would be a checkdep no?

Reply:
The catch2 headers are used during build, not just a checkdep.